repos/coreclr.proj

```diff
@@ -23,6 +23,7 @@
     <BuildArguments>$(BuildArguments) /p:PackagesDir=$(PackagesDir)</BuildArguments>
     <BuildArguments>$(BuildArguments) /p:ContinuousIntegrationBuild=true</BuildArguments>
     <BuildArguments>$(BuildArguments) /p:VersionSuffix="$(PreReleaseVersionLabel)"</BuildArguments>
+    <BuildArguments>$(BuildArguments) /p:UseSharedCompilation=false</BuildArguments>
```

Comment:
This is exactly where I put it to validate locally, too. 👻

Issue tracking getting this enabled (if worthwhile): https://github.com/dotnet/source-build/issues/1473